bin/ksql

```diff
@@ -6,6 +6,9 @@
 #
 
 base_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
```

Comment:
BASH_SOURCE is a special array variable in Bash that contains the source filenames where each corresponding shell function or script is defined. When used in a script, BASH_SOURCE[0] refers to the path of the script being executed. This is particularly useful for determining the directory of the script itself, which can be helpful for setting relative paths.

Reply:
HI @hrishabhg , thank you for the review.

Whilst I understand that the start server works well, I'm talking about the log4j which does not work well and make a noisy exception in the CLI prior to connecting to the server. 

We can avoid it by the suggested check to remove the redundant `/` in case it's not needed.

We can see the exception is outlining the wrong java path.

Reply:
to avoid the followup comment error which I have got as well the above approach was taken.

If you just edit like [#3882 (comment)](https://github.com/confluentinc/ksql/issues/3882#issuecomment-583725482)
```
bin/ksql: line 8: cd: /bin..: No such file or directory
```


Thanks for the merge !